app/src/components/NavSidePanel/NavSidePanelItem.tsx

```diff
@@ -53,13 +54,22 @@ export default defineComponent({
       return !isExternal.value && linkRef?.isActive?.value;
     });
 
+    const isMarginUrl = props.href === MARGIN_FE_URL;
+    let rel = "noopener noreferrer";
+    let target = "_blank";
+
+    if (isMarginUrl) {
+      rel = "noopener";
+      target = "_self";
+    }
+
```

Comment:
think it should not be a part of this PR

Reply:
that has been merged to develop, shouldn't be showing in this diff

Reply:
the margin changes are behind a feature flag